analyzers/tests/SonarAnalyzer.UnitTest/AnalysisContext/SonarAnalysisContextBaseTest.ShouldAnalyzeTree.cs

```diff
@@ -102,8 +102,8 @@ public void ShouldAnalyzeTree_GeneratedFile_ShouldAnalyzeGeneratedProvider_IsCac
         sut.ShouldAnalyzeTree(tree, CSharpGeneratedCodeRecognizer.Instance).Should().BeTrue();
 
         // GetText should be called every time ShouldAnalyzeGenerated is called...
-        additionalText.Verify(x => x.GetText(It.IsAny<CancellationToken>()), Times.Exactly(3));
-        sonarLintXml.ToStringCallCount.Should().Be(1); // ... but we should only try to read the file once
+        additionalText.Verify(x => x.GetText(It.IsAny<CancellationToken>()), Times.Exactly(6)); // TEMPORARILY BUMPED TO 6
```

Comment:
This bump will be reverted once the new SonarLintXmlReader will be used to access all properties.

Reply:
Please add the comment as "ToDo action" because it will raise a n S1135 issue, which we will ignore on this PR, and at the same time it will be present on the feature branch, and it will decrease the possibility of missing it.